go/cmd/dolt/cli/command.go

```diff
@@ -309,6 +309,22 @@ func CheckUserNameAndEmail(config *env.DoltCliConfig) bool {
 	return true
 }
 
+func (hc SubCommandHandler) GetUsage(commandStr string) string {
+	str := "Valid commands for " + commandStr + " are\n"
+
+	for _, cmd := range hc.Subcommands {
+		if hiddenCmd, ok := cmd.(HiddenCommand); ok {
+			if hiddenCmd.Hidden() {
+				continue
+			}
+		}
+
+		str += fmt.Sprintf("    %16s - %s\n", cmd.Name(), cmd.Description())
+	}
+
+	return str
+}
+
 func (hc SubCommandHandler) PrintUsage(commandStr string) {
```

Comment:
(minor) Looks like this function could use the new new `GetUsage` to create the string before it prints it.